x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_eql_search_request.ts

```diff
@@ -89,7 +90,13 @@ export const buildEqlSearchRequest = ({
       runtime_mappings: runtimeMappings,
       timestamp_field: timestampField,
       event_category_field: eventCategoryOverride,
-      tiebreaker_field: tiebreakerField,
+      ...(!isEmpty(tiebreakerField)
+        ? {
+            tiebreaker_field: tiebreakerField,
+          }
+        : {
+            tiebreaker_field: undefined,
```

Comment:
nit: might make sense to spread an empty object here instead of including it with undefined as the value, as there is a subtle difference between the two that could maybe cause an issue sometime in the future?

```
const a = { prop: undefined };
a.prop // undefined
a.hasOwnProperty('prop') // true
'prop' in a // true
for (k in a) {
  console.log(k) // logs 'prop'
}

const b = { };
b.prop // undefined
b.hasOwnProperty('prop') // false
'prop' in b // false
for (k in b) {
   console.log(k) // never hit
}
```

Reply:
Yeah I originally had empty object but changed it to `tiebreaker_field: undefined` in order to keep tests passing. If you think i should do empty object (which it seems you do). i will update the test

Reply:
done https://github.com/elastic/kibana/pull/160082/commits/c63ee6faa7701882b1943cf41574a03882250e23